netmanager/src/views/pages/Dashboard/components/AveragesChart.js

```diff
@@ -24,7 +24,7 @@ import { Bar } from 'react-chartjs-2';
 import domtoimage from 'dom-to-image';
 import JsPDF from 'jspdf';
 import palette from 'theme/palette';
-import axios from 'axios';
+import createAxiosInstance from '../../../apis/axiosConfig';
```

Comment:
Hi @OchiengPaul442 
Thanks again!
Just wondering, do we have a way of using simple paths to minimise mistakes?

1. You could try using `Webpack Module Aliases`?
2. Or `Babel Plugin Module Resolver`?
3. Or `Create React App (CRA)`?
4. Or` ESLint "import/no-unresolved" Rule`?
5. Or `Visual Studio Code Path Intellisense`?
6. etc?